### PR TITLE
Update docs for mainnet

### PR DESCRIPTION
**Pull request by denalimarsh**
- add kava-3 migration guide to docs
- update readme for mainnet
- clone updated readme into docs

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -4,20 +4,27 @@
...
-git checkout v0.3.2
+git checkout v0.8.1
 make install
 ```
 
+### Upgrade
+
+The mainnet upgrade to `kava-3` took place on June 10th, 2020 at 14:00 UTC. The version of Kava for `kava-3` is [__v0.8.1__](https://github.com/Kava-Labs/kava/releases/tag/v0.8.1).
+
+Migration instructions can be found [here](https://docs.kava.io/building/kava-3-migration-guide.html).
```
Comment (fedekunze) [inline docs/readme.md:23]: it says that this file doesn't exist

Comment (rhuairahrighairidh) [reply docs/readme.md:23]: Could you replace it with a relative link like: `./docs/building/kava-3-migration-guide.html`?
github and the link-checker handle these automatically, but not sure about vuepress

Comment (denalimarsh) [reply docs/readme.md]: https://github.com/Kava-Labs/kava/pull/579/commits/accc9b33f3d47604d44120a038a90f990afa75bb

**Comment by denalimarsh** [comment]
There were a couple problems here
- Netlify cache needed to be cleared
- link-check doesn't seem to be validating local URLs against the development build, so they fail the linter despite working in the deploy preview
- link-check didn't like https://ipfs.io/ipfs/QmSYedssC3nyQacDJmNcREtgmTPyaMx2JX7RNkMdAVkdkr/user-growth-fund-proposal.pdf even though it's a valid link (maybe .pdf links aren't valid by default?)

Gonna merge it 👍

**Pull request merged**